infra/terraform/dev/deploy/launch.tf

```
@@ -0,0 +1,35 @@
+data "template_cloudinit_config" "config" {
+  gzip          = false
+  base64_encode = true
+
+  part {
+    content_type = "text/x-shellscript"
+    content      = <<EOT
+#!/bin/bash
+echo ECS_CLUSTER="wikijump-${var.environment}" >> /etc/ecs/ecs.config
```

Since this is a terraform variable and not bash, it should use single quotes:
```suggestion
echo ECS_CLUSTER='wikijump-${var.environment}' >> /etc/ecs/ecs.config
```

---

This actually is bash with some interpolated HCL. It also seems to work as container metrics are flowing correctly in the cluster dashboard so I'd like to leave this alone until it's proven to be a bad idea.

---

I'm going to try and remove the quotes entirely and see if it'll still interpolate correctly.